addons/web/static/src/js/chrome/view_manager.js

```diff
@@ -286,6 +286,7 @@ var ViewManager = Widget.extend(ControlPanelMixin, {
                     controller.appendTo(view.$fragment).done(function() {
                         // Remove the unnecessary outer div
                         view.$fragment = view.$fragment.contents();
+                        view.$fragment.addClass('o_view_manager_content_inner');
```

Comment:
What I was saying before is that we will want to remove the unnecessary controller div. For now, your previous fix was OK. Adding a class on the element will just make it seems important.